docs/release/osg-36.md

```diff
@@ -24,6 +24,17 @@ Known Issues
 
 The following issues are known to currently affect packages distributed in OSG 3.6:
 
+### rrdtool ###
+
+To improve support for Python 3 based GlideinWMS and HTCondor-CE in EL7,
+the EL7 OSG Yum repositories contain a newer version of `rrdtool` than the operating system repositories.
+This may cause dependency solving issues with non-OSG packages.
+Therefore, we suggest adding the following line under the `[osg]` section of `/etc/yum.repos.d/osg.repo`:
```

Comment:
```suggestion
Therefore, on EL7 hosts that are not running GlideinWMS or HTCondor-CE, we suggest adding the following line under the `[osg]` section of `/etc/yum.repos.d/osg.repo`:
```

Reply:
CEs should actually be fine with the OS rrdtool so I'm going to remove the earlier reference to it